st2api/st2api/controllers/v1/aliasexecution.py

```diff
@@ -109,7 +110,8 @@ def _tokenize_alias_execution(self, alias_execution):
     def _extract_parameters(self, action_alias_db, format_str, param_stream):
         formats = []
         for formatstring in action_alias_db.formats:
-            if 'representation' in formatstring:
+            if (isinstance(formatstring, collections.Mapping) and
+               'representation' in formatstring and formatstring['representation']):
```

Comment:
`and formatstring('representation', None)` is slightly shorter and has the same effect :)